README.md

```diff
@@ -30,12 +31,19 @@ most important part. Add your answer to this markdown file.
 
 Answer:
 
-T(n) = {$\Theta$(0)   if n = 0
+T(n) = {
+$\Theta$(0)   if n = 0
 
 $\Theta$(1)   if n = 1
 
 $\Theta$(2)   if n = 2
 
 3T(n/3) + $\Theta$(n)    if n < 2
```

Comment:
According to your code, the recurrence relation is not correct. The combined part is not Theta of n.